### ISSUE TITLE
code: Add configuration file for consensus parameters

### ISSUE THREAD

**Issue by ancazamfir** [circlefin/malachite#134]
Add configuration file for consensus parameters. It should include:
- a subset of the parameters we have for Comet:
```
wal_file = "data/cs.wal/wal"

# How long we wait for a proposal block before prevoting nil
timeout_propose = "3s"
# How much timeout_propose increases with each round
timeout_propose_delta = "500ms"
# How long we wait after receiving +2/3 prevotes for “anything” (ie. not a single block or nil)
timeout_prevote = "1s"
# How much the timeout_prevote increases with each round
timeout_prevote_delta = "500ms"
# How long we wait after receiving +2/3 precommits for “anything” (ie. not a single block or nil)
timeout_precommit = "1s"
# How much the timeout_precommit increases with each round
timeout_precommit_delta = "500ms"
```

- the configuration for L55 `f+1` (default) or something else in the [`f+1`, `2f+1`] range

Note: 
- the `delta` parameters require timeout value increases across rounds.
- in Comet we also have the `timeout_commit`. I think we can do this in a separate PR as it requires more extensive updates.
```
# How long we wait after committing a block, before starting on the new
# height (this gives us a chance to receive some more precommits, even
# though we already have +2/3).
timeout_commit = "5s"
```
